backend/Makefile

```diff
@@ -56,7 +56,7 @@ docker-first-run:
 	docker compose run web bash -c '\
 		python manage.py makemigrations &&\
 		python manage.py migrate &&\
-		python manage.py create_views \
+		python manage.py create_views --api_version "api_v1_0_0_beta"\
```

Comment:
I didn't add the `create_views` function here because it's probably better to do this in one place.

Reply:
Maybe `create_views` just creates every versioned view _unless_ it's passed an `--api_version` flag for development/testing purposes.

Reply:
That's a good idea. Note that this all wants for a revision. I slammed the API versioning in pretty quick.